### PR TITLE
Calorie progress bar hook up to global context data

### PR DESCRIPTION
**Pull request by mmoren01**

### CODE REVIEW

File: _tests_/front-end/components/calorieTrscker.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import {render} from '@testing-library/react';
```
Comment (Scott-Guinn) [inline _tests_/front-end/components/calorieTrscker.test.js]: I think there's a typo in this file name

File: components/Apple.jsx [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
 const Apple = () => {
+  const {theme} = useContext(Context);
+  const [appleBgColor, setAppleBgColor] = useState('rgb(255, 255, 255)');
+  useEffect(() => {
+    if (theme === 'dark') {
+      setAppleBgColor('rgb(52, 58, 64)');
+    } else if (theme === 'light') {
+      setAppleBgColor('rgb(255, 255, 255)');
+    }
+  }, [theme]);
```
Comment (Scott-Guinn) [inline components/Apple.jsx:13]: Nice, thanks for implementing this!

**Review by Scott-Guinn — APPROVED** [review]
Looks great. Only change is the typo in the test filename

**Review by nohren — APPROVED** [review]
nice work!

File: _tests_/front-end/components/calorieTrscker.test.js (hunk 1 above)
Comment (Scott-Guinn) [reply _tests_/front-end/components/calorieTrscker.test.js]: I took care of this.

**Pull request merged**